gem/templates/forgot_password.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+    <div class="default-container">
+        <div class="title">
+            <h3>{% trans "Forgot PIN" %}</h3>
+        </div>
+        <div class="block">
+            <form method="post">
+                {% csrf_token %}
+
+                {% if form.errors %}
+                    <p class="error errorlist">
+                    {% for key, errors in form.errors.iteritems %}
+                        {% for error in errors %}
+                            {% trans error %}<br/>
+                        {% endfor %}
+                    {% endfor %}
+                    </p>
+                {% endif %}
+
+                <p><strong>{% trans "Username" %}</strong></p>
+                {{ form.username }}
+
+                <p><strong>{% trans "Security question" %}</strong></p>
+                <p>{% trans "Please answer the following security question." %}</p>
+
+                <p><strong>{% trans random_security_question %}</strong></p>
+                {{ form.random_security_question_answer }}
+
+                <br>
+                <input type="submit" value='{% trans "FORGOT PIN" %}' />
```

Comment:
I think "FORGOT" needs to be title case to match `line 7`. Any upper casing should be handled in the CSS if needed.